drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/RightPanelViewImpl.html

```diff
@@ -100,8 +100,31 @@
         <!-- content for cheatsheet tab -->
         <div id="kieScenarioCheatsheetTabContent"
              class="tab-pane fade"
-             aria-labelledby="Scenario Cheatsheet">
-            <p>Create a row for each scenario, and input values using the following syntax.</p>
+             aria-labelledby="Scenario Cheatsheet" style="padding: 1em">
+            <p>Use contextual menu to add/remove columns and rows</p>
```

Comment:
Please consider putting these sentences into a numbered list (as the text is a kind of a procedure) and ending them with a period.

Reply:
Ok for the period but I don't agree about the numbered list because it is not a step by step guide, user can follow any order he prefers.
 @gadeyredhat WDYT?

Reply:
@danielezonca , ideally if this is a procedure, it should be numbered but as you say its not step by step you can skip numbering but atleast use bullets for each point.